package-lock.json

```diff
@@ -434,6 +445,17 @@
             "lodash": "^4.17.13",
             "to-fast-properties": "^2.0.0"
           }
+        },
```

Comment:
Something seems wrong here
There are 3x `chalk` entries and this PR doesn't even have anything to do with `chalk`